tests/test_pmi_crud.py

```diff
@@ -0,0 +1,149 @@
+import pry
```

Comment:
Thanks for the happy path testing! I look forward to seeing the sad path testing